cpp/lib/src/MARC.cc

```diff
@@ -988,10 +988,14 @@ std::string Record::getPublicationYear(const std::string &fallback) const {
     const auto _008_field(findTag("008"));
     if (likely(_008_field != end())) {
         const auto &field_contents(_008_field->getContents());
-        if (likely(field_contents.length() >= 12)) {
-            const std::string year_candidate(field_contents.substr(7, 4));
-            if (ConsistsOfDigitsOnly(year_candidate) and year_candidate != "9999")
-                return year_candidate;
+        if (likely(field_contents.length() >= 16)) {
+            const std::string year_candidate(field_contents.substr(11, 4));
```

Comment:
Are you sure this is 11-15? Here are some examples with 7-10:
- https://ixtheo.de/Record/1049156137#details
- https://ixtheo.de/Record/1725646374#details